src/views/ModelPlan/ReadOnly/_components/FilterView/Banner/index.tsx

```diff
@@ -54,7 +56,7 @@ const FilterViewBanner = ({
                 type="button"
                 unstyled
                 className="text-white text-no-wrap"
-                onClick={() => history.push(`${history.location.pathname}`)}
+                onClick={() => history.push(history.location.pathname)}
```

Comment:
Was this a functional change, or just stylistic? Only case I could see this mattering is pushing `undefined` or `null` vs `"undefined"` or `"null"`